src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -0,0 +1,1160 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.esotericsoftware.kryo.DefaultSerializer;
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+import com.google.cloud.dataflow.sdk.options.PipelineOptions;
+import com.google.common.annotations.VisibleForTesting;
+import htsjdk.samtools.*;
+import org.apache.spark.HashPartitioner;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.api.java.function.VoidFunction;
+import org.apache.spark.broadcast.Broadcast;
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.SparkProgramGroup;
+import org.broadinstitute.hellbender.engine.spark.GATKSparkTool;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.spark.utils.HopscotchHashSet;
+import org.broadinstitute.hellbender.tools.spark.utils.MapPartitioner;
+import org.broadinstitute.hellbender.utils.gcs.BucketUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.ReadUtils;
+import scala.Tuple2;
+
+import java.io.*;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.function.Function;
+
+/**
+ * Tool to describe reads that support a hypothesis of a genomic breakpoint.
+ */
+@CommandLineProgramProperties(summary="Find reads that evidence breakpoints.",
+        oneLineSummary="Dump FASTQs for local assembly of putative genomic breakpoints.",
+        programGroup = SparkProgramGroup.class)
+public final class FindBreakpointEvidenceSpark extends GATKSparkTool {
+    private static final long serialVersionUID = 1L;
+    private static final int MIN_MAPQ = 20;
+    private static final int MIN_MATCH_LEN = 45; // minimum length of matched portion of an interesting alignment
```

Comment:
It looks like it wasn't ported over from GATK 3 yet, here it is in that codebase:

https://github.com/broadgsa/gatk/blob/0b73e380436aaa5a41fb3aab97ab651207669f47/public/gatk-engine/src/main/java/org/broadinstitute/gatk/engine/filters/OverclippedReadFilter.java

There's also a version in htsjdk: `htsjdk.samtools.filter.OverclippedReadFilter`, but that's hard to use because it wants a SamRecord rather than a GATKRead. We should probably just port it over into GATK4.